calico/felix/fetcd.py

```diff
@@ -242,18 +243,21 @@ def watch_etcd(self, update_splitter):
                 profile_id, rules = parse_if_rules(response)
                 if profile_id:
                     _log.info("Scheduling profile update %s", profile_id)
-                    update_splitter.on_rules_update(profile_id, rules)
+                    update_splitter.on_rules_update(profile_id, rules,
+                                                    async=False)
```

Comment:
I assume async=False sticks to existing behavior?  Is this as you'd expect though?